### ISSUE TITLE
Schedules with courses on separate campuses 

### ISSUE THREAD

**Issue by InsanesTheName** [ComputerScienceHouse/schedulemaker#81]
I've come across an issue where, in looking at potential schedules, certain sections of a class not on the RIT Main campus are appearing (and being included in) potential schedules with RIT Main classes, which is obviously irrational. I ran across this issue with CSEC-101; section 06 of this class is on the Dubai campus according to SIS.

**Comment by bgrawi** [comment]
Yeah, I've known about this for a while. It's funny because I had someone from Dubai want the exact opposite! So yes, a location filter would be a helpful!

**Comment by devinmatte** [comment]
@bgrawi Do we currently handle this now? This is a  fairly old issue so I wanted to see if this needs to be handled or not?
